<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="top_down_render" name="refine_map" type="refine_map" output="screen">
    <param name="sem_occ_grid_path" value="/home/ian/code/PanoPoints/build/atlas_sem_grid.vtk"/>
    <param name="original_map_path" value="$(find top_down_render)/maps/ucity"/>
    <param name="map_center_x" value="0"/>
    <param name="map_center_y" value="0"/>
  </node>
</launch>
